api/lib/block-listeners/cosmos-node-subscription.js

```diff
@@ -204,20 +204,20 @@ class CosmosNodeSubscription {
   }
 
   async addPopularityToValidators(validators, networkId) {
-    return Promise.all(
-      validators.map(async (validator) => {
-        // popularity is actually the number of views of a validator on their page
-        const popularity = await this.db.getValidatorViews(
-          validator.operatorAddress,
-          networkId
-        )
-        // we add the popularity field to the validator
-        return {
-          ...validator,
-          popularity: popularity || 0
-        }
-      })
-    )
+    // popularity is actually the number of views of a validator on their page
+    const validatorPopularity = await this.db.getValidatorsViews(networkId)
```

Comment:
make this a dictionary to speed up the search for each validator.

Reply:
Done